consul/datadog_checks/consul/consul.py

```diff
@@ -190,7 +190,7 @@ def _check_for_leader_change(self, instance, instance_state):
                     instance_state.last_known_leader, leader))
 
                 self.event({
-                    "timestamp": int(datetime.now().strftime("%s")),
+                    "timestamp": int(datetime.now().timestamp()),
```

Comment:
This only works with Python >= 3.3 and it doesn't seem to be any official backport, I can't merge this patch as it is, sorry.